cache/clock_cache.cc

```diff
@@ -825,6 +827,11 @@ FixedHyperClockTable::HandleImpl* FixedHyperClockTable::Lookup(
           // Acquired a read reference
           if (h->hashed_key == hashed_key) {
             // Match
+            // Update the hit bit
+            if (eviction_callback_) {
```

Comment:
I proposed this optimization where we only update the hit marker if there is an eviction callback. However, assuming we keep that optimization, I think it would be better to have the hit boolean as an eviction handler parameter than to have the getter function on Cache, which seems to be inviting general use regardless of eviction handler being present.